lib/active_merchant/billing/gateways/pay_trace.rb

```diff
@@ -159,6 +165,22 @@ def acquire_access_token
 
       private
 
+      # method can only be used to add level 3 data to any approved and unsettled sale transaction so it is built into the standard purchase workflow above
```

Comment:
can `level_3` data be added to a capture?

Reply:
It can! And now it has been! 🎉